js/src/util/utf8.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from 'text-encoding-utf-8';
 
 /** @ignore @suppress {missingRequire} */
-const _Buffer = typeof Buffer === 'function' ? Buffer : null;
+const _Buffer = eval("typeof Buffer === 'function' ? Buffer : null");
```

Comment:
This will cause problems in environments with a CSP that does not allow eval, no?

Reply:
I guess we are already using `new Function` so that ship has sailed?

Reply:
Oh I forgot about CSPs. I'll revert this for now, and change the webpack configs to specify `{ node: { buffer: false } }`.

Haven't gotten any complaints about using the Function constructor, but https://github.com/webpack/webpack/issues/5627 seems to have a few workarounds.

Reply:
It would be good to try. Maybe it's okay if the function constructor never gets called. 

I'm Vega, we added an ast evaluator in addition to a function constructor to support expressions in environments with CSP.